arangod/Agency/MoveShard.cpp

```diff
@@ -55,7 +56,8 @@ MoveShard::MoveShard(Node const& snapshot, AgentInterface* agent, std::string co
       _from(id(from)),
       _to(id(to)),
       _isLeader(isLeader),  // will be initialized properly when information known
-      _remainsFollower(isLeader) {}
+      _remainsFollower(isLeader),
+      _toServerIsFollower(false) {}
```

Comment:
How about initializing this member inline at the definition? ATM the 3rd constructor (2 lines down) is still not initializing `_toServerIsFollower`.